server/routerlicious/packages/lambdas/src/scribe/lambdaFactory.ts

```diff
@@ -135,10 +141,13 @@ export class ScribeLambdaFactory extends EventEmitter implements IPartitionLambd
         let expectedSequenceNumber = lastCheckpoint.protocolState.sequenceNumber + 1;
         for (const message of opsSinceLastSummary) {
             if (message.sequenceNumber !== expectedSequenceNumber) {
-                await this.sendLambdaStartResult(tenantId, documentId, {lambdaName: LambdaName.Scribe, success: false});
-                throw new Error(`Invalid message sequence from checkpoint/summary.`
+                const error = new Error(`Invalid message sequence from checkpoint/summary.`
                     + `Current message @${message.sequenceNumber}.`
                     + `Expected message @${expectedSequenceNumber}`);
+                scribeSessionMetric?.error("Invalid message sequence from checkpoint/summary", error);
```

Comment:
Do we want to also add the sequenceNumbers (current and expected) to the message field when we `.error()` the `scribeSeccionMetric`? The `error` object will be saved in the `scribeSessionMetric`, but I wonder if later, when we look at telemetry data, we would be paying more attention to the "message" property instead of the "error" property.

Since the `error` object on line 144 has the same "message" but with the seq number info, one alternative would be to create an error string with the full message. Then, we pass the string both to `scribeSessionMetric?.error()` and to a new Error that we would throw at line 150.

Reply:
Spoke to Henrique offline: The idea here is to have the Message be generic so that we can summarize on it to get an accurate count of how often it might be happening. For the stacktrace and other specific information, I have been passing it to the error property